apps/files/src/default.js

```diff
@@ -34,37 +38,37 @@ const appInfo = {
       },
       quickAccess: {
         icon: 'share',
-        ariaLabel: 'Collaborators'
+        ariaLabel: $gettext('Collaborators')
       }
     }
   ]
 }
 const navItems = [
   {
-    name: 'All files',
+    name: $gettext('All files'),
     iconMaterial: appInfo.icon,
     route: {
       name: 'files-list',
       path: '/'
     }
   },
   {
-    name: 'Favorites',
+    name: $gettext('Favorites'),
     iconMaterial: 'star',
     route: {
       name: 'files-favorites'
     }
   },
   {
-    name: 'Shared with others',
+    name: $gettext('Shared with others'),
     iconMaterial: 'share',
     route: {
       name: 'files-shared-with-others',
       path: `/${appInfo.id}/shared-with-others`
     }
   },
   {
-    name: 'Deleted files',
+    name: $gettext('Deleted files'),
```

Comment:
Do we need the `$gettext` here when you use it then again in components?

Reply:
it is needed here to get the string extracted. the real translation work is done in the component